### DISCUSSION TITLE
纯小白求指教：ValueError: Axis limits cannot be NaN or Inf

### DISCUSSION THREAD

**Discussion by love530love** [babysor/MockingBird#766] [Q&A]
python demo_toolbox.py运行程序后，导入语音，合成罗输出时报错ValueError: Axis limits cannot be NaN or Inf
![QQ截图20221015215152](https://user-images.githubusercontent.com/92354089/195990143-b7b1633c-31b3-44c1-96ec-983a488d947a.png)

**Comment by babysor** [comment]
窗口太小渲染不了，拖大一点就好

**Reply by love530love** [reply]
非常感谢，果真按你说的拖大窗口或全屏后就不再报此错误了！
